### PR TITLE
tweaked install UI

### PR DESCRIPTION
**Pull request by jasmingacic**
Signed-off-by: jasmingacic <jasmin.gacic@gmail.com>
image

### CODE REVIEW

**Review by donnalux — DISMISSED** [review]
Looks great!

File: cmd/kusk/cmd/deploy.go
Comment (kylehodgetts) [inline cmd/kusk/cmd/deploy.go:196]: Print `e` if it's nil?

**Review by kylehodgetts — CHANGES_REQUESTED** [review]
Will approve once all the commented out code is removed

File: cmd/kusk/cmd/deploy.go
Comment (jasmingacic) [reply cmd/kusk/cmd/deploy.go:196]: duh :D

File: cmd/kusk/cmd/install.go
Comment (kylehodgetts) [inline cmd/kusk/cmd/install.go:168]: Needed?

File: cmd/kusk/cmd/install.go
Comment (kylehodgetts) [inline cmd/kusk/cmd/install.go:162]: Needed?

Comment (jasmingacic) [reply cmd/kusk/cmd/install.go]: no

File: cmd/kusk/cmd/install.go
Comment (jasmingacic) [reply cmd/kusk/cmd/install.go]: nope

**Review by kylehodgetts — APPROVED** [review]

**Comment by jasmingacic** [comment]
@donnalux one more look before i merge this 
image

**Pull request merged**
